pkg/anchore/client.go

```diff
@@ -183,7 +183,7 @@ func getImageDigest(imageRef string, isCached bool) (string, error) {
 }
 
 //CheckImage checking Image with Anchore
-func CheckImage(image string, isCached bool) (v1alpha1.AuditImage, bool) {
+func CheckImage(image string, isCached bool, isWl bool) (v1alpha1.AuditImage, bool) {
```

Comment:
I would use a full name here instead of an abbreviation.

Reply:
I'll fix it